internal/test/wsu/wsu_test.go

```diff
@@ -322,6 +308,35 @@ func testEastWestNetworking(t *testing.T) {
 	// TODO: test Windows <-> Windows on different node
 }
 
+// deployWindowsServer creates a deployment with a single Windows Server pod, listening on port 80
+func deployWindowsServer() (*appsv1.Deployment, error) {
```

Comment:
Please rename to `deployWindowsWebServer()` to indicate the type of Windows application. `WindowsServer` makes me think it is just a Windows Server.